samples/build/build_buildpacks-v3_cr.yaml

```diff
@@ -1,14 +1,15 @@
 apiVersion: build.dev/v1alpha1
 kind: Build
 metadata:
-  name: example-build-buildpack
+  name: buildpacks
 spec:
-  # Add fields here
   source:
     url: https://github.com/sclorg/nodejs-ex
-  strategy: 
-    name: "buildpacks-v3"
-    namespace: "openshift"
-  builderImage: "cloudfoundry/cnb:bionic"
```

Comment:
Both `cloudfoundry/cnb:bionic` and `heroku/buildpacks:18` are implementing buildpacks.io lifecycle specification, therefore, both are first class citizens in this example.